src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,51 @@
+<%@ page import="java.util.List" %>
+<%@ page import="codeu.model.data.Conversation" %>
+<%@ page import="codeu.model.data.Message" %>
```

Comment:
Do you think we need to import Conversations, Messages and Lists into the profile.jsp?

Reply:
Ooh you're right, I imported them just in case, but I ended up not using them. I'll remove those - thanks!!